src/main/java/org/linkeddatafragments/solver/ReorderTransformationLDF.java

```diff
@@ -61,7 +61,6 @@ private void initializeMatcher() {
         matcher.addPattern(new Pattern(numTriples, VAR, VAR, VAR));     // ???
     }
 
-    @Override
```

Comment:
in 2.13.0 there is no ReorderTransformationBase but the class was renamed to ReorderTransformationSubstitution